baker/siteRenderers.tsx

```diff
@@ -414,14 +423,99 @@ export const countryProfileCountryPage = async (
 
 export const flushCache = () => getCountryProfilePost.cache.clear?.()
 
+const renderPostThumbnailBySlug = async (
+    slug: string | undefined
+): Promise<string | undefined> => {
+    if (!slug) return
+
+    let post
+    try {
+        post = await getPostBySlug(slug)
+    } catch (err) {
+        // if no post is found, then we return early instead of throwing
+    }
+
+    if (!post?.thumbnailUrl) return
+    return ReactDOMServer.renderToStaticMarkup(
+        <img src={formatUrls(post.thumbnailUrl)} />
+    )
+}
+
+export const renderProminentLinks = async ($: CheerioStatic) => {
+    const blocks = $("block[type='prominent-link']").toArray()
+    await Promise.all(
+        blocks.map(async (block) => {
+            const $block = $(block)
+            const formattedUrlString = $block.find("link-url").text() // never empty, see prominent-link.php
+            const formattedUrl = Url.fromURL(formattedUrlString)
+
+            const resolvedUrl = await resolveInternalRedirect(formattedUrl)
+            const resolvedUrlString = resolvedUrl.fullUrl
+
+            const style = $block.attr("style")
+            const content = $block.find("content").html()
+
+            let title
+            try {
+                title =
+                    $block.find("title").text() ||
+                    (!isCanonicalInternalUrl(resolvedUrl)
+                        ? null // attempt fallback for internal urls only
+                        : resolvedUrl.isExplorer
+                        ? await getExplorerTitleByUrl(resolvedUrl)
+                        : resolvedUrl.isGrapher && resolvedUrl.slug
+                        ? (await Chart.getBySlug(resolvedUrl.slug))?.config
+                              ?.title // optim?
+                        : resolvedUrl.slug &&
+                          (await getPostBySlug(resolvedUrl.slug)).title)
+            } finally {
+                if (!title) {
+                    logContentErrorAndMaybeSendToSlack(
+                        new Error(
+                            `No fallback title found for prominent link ${resolvedUrlString}. Block removed.`
+                        )
+                    )
+                    $block.remove()
+                    return
+                }
+            }
+
+            const image =
+                $block.find("figure").html() ||
+                (!isCanonicalInternalUrl(resolvedUrl)
+                    ? null
+                    : resolvedUrl.isExplorer
+                    ? renderExplorerDefaultThumbnail()
+                    : resolvedUrl.isGrapher && resolvedUrl.slug
+                    ? await renderGrapherImageByChartSlug(resolvedUrl.slug)
```

Comment:
Not sure if authors are happy with this if it contains query parameters? (given that we recently [got rid of social media thumbnails](https://github.com/owid/owid-grapher/issues/1086))

But something that we could easily change later, so wouldn't block this PR.

Reply:
@JoeHasell Daniel rightly points out that some of the grapher prominent links thumbnails will be out of sync (given that query parameters, such as e.g. `?country=~USA` will be ignored). This is already the case now.
Our options:
- (1) replace any grapher thumbnail with the default grapher thumbnail introduced by @danielgavrilov 
- (2) leave as is

Given:
- the size of those thumbnails, where accuracy cannot be evaluated
- the upcoming sunsetting of the large prominent link variation

I'd weakly be in favour of keeping them as-is, as they still give an indication of what to expect without requiring a click. Thoughts?

Reply:
Yes I agree @mlbrgl. In the short-term, leaving as it is now is probably the better of those two options. Having the default image isn't as 'wrong' as it is in the social media thumbnail case, and it does convey at least some information (even just to us). 

Longer-term we can revisit this question again in the wider context of entry redesign.